debugger.md

```diff
@@ -0,0 +1,83 @@
+* If you don’t have mono installed you won’t get IntelliSense. See optional installation section below.
+* **Hint:** Setup for Debugging CLR on OSX changed fundamentally end of February 2016. For existing projects please make sure to update your launch.json file according to these steps.​
+
+###First Time setup
+1. Get Visual Studio Code
+    * Install Visual Studio Code. Pick the latest insider version from here: https://code.visualstudio.com/insiders
+    * **Hint:** While we are in a pre-release phase vscode might ask you to install a C# extension from the marketplace. DO NOT INSTALL IT! This extension doesn't contain debugger bits yet and might bring your system in a weird state.   
+2. Install Dotnet CLI
+ * Install Dotnet CLI following the instructions here:  http://dotnet.github.io/getting-started  
+ * **Hint for Linux**:  There's one point where you have to do things different - we need a later version of dotnet: 
+   (So don't run: *sudo apt-get install dotnet=1.0.0.001598-1* but instead run *sudo apt-get install dotnet-nightly*)
+ * **Hint for Mac**: Dotnet CLI requires openSSL to work. It will ask you to install this during installation. Don't forget this!
+3. C# Extension for VS Code
+ * Get the .vsix file from here: \\vsdbgqa\x-plat\vscode-extension\csharp
+ * Open "omnisharp-0.3.2.vsix” from within VSC to install the extension. VSC will show a message that the extension has been installed and it will restart. 
+ * **Magic step:** After the extension is installed, open a C# file. VSC might ask you to install "C#". Select to NOT install it! 
+ * In the background a process is triggered to get required bits. You can follow that process in the output window of VSC. Wait to finish that process.
+4. Install Mono 
+ * *Currently not required due to issue with command to create tasks.json*
+ * ~~To be able to auto-create a tasks.json for every project you'll currently need Mono. You could skip this step, but then you need to create the file manually.~~
+ * ~~Run the following commands in a terminal:~~
+ * ~~On Mac: brew install mono~~
+ * ~~On Linux: sudo apt-get install mono-complete~~
+
+###Once for each project
+The following steps have to executed for every project. 
+* Whenever you want to start debugging (e.g. by pressing the debugger-play button) a .NET Core app select '.NET Core"  for debug environment in the command pallette when VS code first asks for it. 
+* This will create a launch.json file. Enter the name of the executable in 'program' field (including the path). This could be something like *"${workspaceRoot}/bin/Debug/dnxcore50/osx.10.11-x64/HelloWorld"*. 
+* For attach scenarios modify the processName attribute of launch.json
+* If you want 'compile' support for F5 you need to create a tasks.json file . There are multiple ways to do this.
+  * You can get it here (\\vsdbgqa\x-plat\SampleApp\cli\.vscode\tasks.json) and put it manually next to your launch.json file 
```

Comment:
I would suggest pointing people to the website -- https://github.com/OmniSharp/omnisharp-vscode/blob/dev/template-tasks.json

If you want to leave it on \\\\vsdbgqa, you need to double your back slashes.